docs/UserGuide.md

```diff
@@ -266,12 +262,39 @@ Examples:
 [Back to top](#table-of-contents)<br>
 [Back to Project Glossary](#project-glossary)
 
+--------------------------------------------------------------------------------------------------------------------
+### **Deleting a Project :** `delproj`
+
+Deletes the Project at the specified `INDEX` from the Project list.
+
+Format: `delproj INDEX`
+
+```yaml
+Note:
+  * INDEX refers to numbering of the Projects shown in the displayed Project list.
+  * INDEX must be a positive integer 1, 2, 3, …​
+  * Staff list will not display any Staff by default. You will have to use the [view command](#view-the-staff-list-within-a-project--view) to see the Staff of that Project.
+```
+
+Examples:
+* `list` followed by `delproj 2` deletes the 2nd Project in the displayed list of Projects.
+* `findproj 2103T TP` followed by `delproj 1` deletes the 1st Project in the results of the `findproj` command.
+
+[Back to top](#table-of-contents)<br>
+[Back to Project Glossary](#project-glossary)
+
 --------------------------------------------------------------------------------------------------------------------
 ## **Staff Commands**
 
 Since each Project contains a list of Staff, we recommend that before using Staff commands on any Project, 
 you should first use the [view command](#view-the-staff-list-within-a-project--view) to display the Project's attached Staff list.
 
+```yaml
+Warning !!!:
+  * Failing to do so may result in the Staff commands not working as intended.
+```
```

Comment:
May want to explain more on this? Otherwise it may not be helpful to include this warning here.